README.md

```diff
@@ -204,6 +210,47 @@ public class MainApplication extends Application implements ReactApplication {
 }
 ```
 
+###### **Kotlin:**
+
+```kotlin
```

Comment:
```kotlin
 // MainApplication.kt
import com.adobe.marketing.mobile.Edge
import com.adobe.marketing.mobile.Lifecycle
import com.adobe.marketing.mobile.LoggingMode
import com.adobe.marketing.mobile.MobileCore
import com.adobe.marketing.mobile.MobileCore.getApplication
import com.adobe.marketing.mobile.edge.consent.Consent
import com.adobe.marketing.mobile.edge.identity.Identity
 ```

 ```kotlin
// MainApplication.kt
 class MainApplication : Application(), ReactApplication {
   ...
 override fun onCreate() {
    super.onCreate()
    
    MobileCore.setApplication(this);
    MobileCore.setLogLevel(LoggingMode.DEBUG)
    MobileCore.configureWithAppID("YOUR-APP-ID");
    MobileCore.registerExtensions(
      listOf(
        Lifecycle.EXTENSION,
        Edge.EXTENSION,
        Identity.EXTENSION,
        Consent.EXTENSION
      ),
    ) {
      Log.d("MainApp", "Adobe Experience Platform Mobile SDK was initialized")
    }

    SoLoader.init(this, false)
    if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
      // If you opted-in for the New Architecture, we load the native entry point for this app.
      load()
    }
    ApplicationLifecycleDispatcher.onApplicationCreate(this)
  }

// Enable Lifecycle Start (application.launch)
fun onResume() {
    MobileCore.setApplication(getApplication()!!)
    MobileCore.lifecycleStart(null)
  }

// Enable Lifecycle Pause (application.close)
  fun onPause() {
    MobileCore.lifecyclePause()
  }
 ```
```kotlin
// MainActivity.kt

// Called when the main activity start interacting
        super.onResume()
        (application as MainApplication).onResume()
    }

// Called when the activity pauses
 override fun onPause() {
        super.onPause()
        (application as MainApplication).onPause()
    }
```

Reply:
works fine have tested it @cacheung @ishita-gambhir-adobe

Reply:
@namArora3112 Update the Android lifecycle with global lifecycle callbacks in MainActivity.kt , can you try in your expo app to confirm?

Reply:
Just need to add these imports at the top for Activity, Application,

`import android.app.Activity
import android.app.Application`
This should cover all references used.

Reply:
Thanks @namArora3112 , I updated the import for the MainActivity.kt.  
I don't see `import android.app.Activity` is needed for MainApplication.kt.   And `import android.app.Application` was there by default when creating the file, so I didn't include it for SDK setup.

Please let me know if there is any question.

@ishita-gambhir-adobe,  you can incorporate these review comments to the doc now.  Thanks.